src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <div class="search-box">
+        <div class="search-icon">
+          <img
+            class="search-logo"
+            src="./images/Search.svg"
+            alt="Search logo"
+          >
+        </div>
+        <div class="div-input">
+          <input
+            class="search-input"
+            type="text"
+            name="search-input"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </div>
+      </div>
+    </form>
+
+    <form
+      class="search-form search-form--small"
+      action="/"
+      method="get"
+      data-qa="small"
+    >
+      <div class="search-box search-box--small">
+        <div class="search-icon search-icon--small">
+          <img
+            class="search-logo search-logo--small"
+            src="./images/Search.svg"
+            alt="Search logo"
+          >
+        </div>
+        <div class="div-input div-input--small">
+          <input
+            class="search-input search-input--small"
+            type="text"
+            name="search-input"
+            placeholder="Los Angeles"
```

Comment:
Main reason your tests are failed
```suggestion
            placeholder="Try &quot;Los Angeles&quot;"
```